modules/oe_theme_content_person/src/Plugin/ExtraField/Display/PersonJobListExtraField.php

```diff
@@ -108,15 +109,24 @@ public function viewElements(ContentEntityInterface $entity) {
         $body = $view_builder->viewField($person_job->get('oe_description'), [
           'label' => 'hidden',
         ]);
-        $job_list['#simple'] = FALSE;
+      }
+      else {
+        // Don't add an item if Person Job has an empty description field.
+        $cacheable_metadata->addCacheableDependency($person_job);
+        continue;
       }
       $job_list['#items'][] = [
-        'label' => $person_job->label(),
+        'label' => $single_person_job ? NULL : $person_job->label(),
         'body' => $body,
       ];
       $cacheable_metadata->addCacheableDependency($person_job);
     }
-
+    // Hide the job list if there are no items.
+    if (empty($job_list['#items'])) {
+      $build = [];
```

Comment:
We need to use the same variable as above, i.e. `$isEmpty`.
Please remove the `$body` initialisation from line 107 because we don't need it.